public/index.html

```diff
@@ -0,0 +1,375 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no">
+  <title>Ballers: Become A Landlord</title>
+  <link rel="apple-touch-icon" sizes="180x180" href="./apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="./favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="./favicon-16x16.png">
+  <link rel="manifest" href="./site.webmanifest">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="./assets/slick/slick.css"/>
+  <link rel="stylesheet" type="text/css" href="./assets/slick/slick-theme.css"/>
+  <link rel="stylesheet" type="text/css" href="assets/css/ballers.css">
+
+</head>
+
+<body>
+  <!-- Holding -->
+  <section class="holding">
+
+    <!-- Header -->
+    <header>
+      <nav class="navbar navbar-expand-lg">
+        <a class="navbar-brand" href="#"><img class="ballers-logo" src="./assets/img/ballers-logo.png" alt=""></a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+      
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav mr-auto">
+            <li class="nav-item active">
+              <a class="nav-link" href="#">A-Z of BALL</span></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">About Us</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Contact Us</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">FAQ</a>
+            </li>
+          </ul>
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <a class="nav-link sign-in" href="#">Sign In</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link register btn" href="#">Register</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!-- /.Header -->
+
+    <section class="holding-inner">
+      <h1 class="holding-heading">Become <br> a landlord</h1>
```

Comment:
Not compulsory now, however, if you used BEM as advised, I believe the naming will be easier to read and developer-friendly. This will be `holding-inner__heading`, and the below text will be `holding-inner__text`